.github/workflows/server.yml

```diff
@@ -185,6 +185,9 @@ jobs:
     timeout-minutes: 15
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
why do we need `fetch-depth: 0`? That will make the checkouts slower and I don't think we need that ... do we?

Reply:
We don't need it. I was confused about its meaning. I'll remove it.